kart/lfs_util.py

```diff
@@ -61,6 +110,8 @@ def get_hash_from_pointer_file(pointer_file_bytes):
 
 def get_local_path_from_lfs_hash(repo, lfs_hash):
     """Given a sha256 LFS hash, finds where the object would be stored in the local LFS cache."""
+    if lfs_hash.startswith("sha256:"):
+        lfs_hash = lfs_hash[len("sha256:") :]
```

Comment:
```suggestion
        lfs_hash = lfs_hash[len("sha256:"):]
```

Or:

```suggestion
        lfs_hash = lfs_hash[7:]  # len("sha256:")
```

Reply:
Went with the second. Black doesn't like the first one